Gekko/UI/MainViewController.swift

```diff
@@ -577,7 +581,18 @@ typealias CompletionHandler = () -> Void
         self.currenciesController.collectionView!.reloadData()
     }
     
-    fileprivate func setupServerErrorView() {
+    fileprivate func scheduleServerAccessibility() {
```

Comment:
Please move this method inside AccessibilityController and call it startMonitoringAccessibility